### PR TITLE
Q metric refactored after memorydb migration

### PR DESCRIPTION
**Pull request by p-fernandez**
### What change does this PR introduce?

Introduces the new metrics queues services initiated by @Cliftonz but implemented after the refactoring made with the MemoryDB migration.

### Why was this change needed?

To align the codebase as the deviation was huge.

### Other information (Screenshots)

Co-Author: Zac Clifton <zac@novu.co>

### CODE REVIEW

File: packages/application-generic/src/health/completed-jobs-metric-queue.health-indicator.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  HealthIndicatorResult,
+} from '@nestjs/terminus';
+import { Injectable, Logger } from '@nestjs/common';
+
+import { CompletedJobsMetricQueueService } from '../services';
+
+const LOG_CONTEXT = 'CompletedJobsMetricQueueServiceHealthIndicator';
+
+@Injectable()
+export class CompletedJobsMetricQueueServiceHealthIndicator extends HealthIndicator {
```
Comment (LetItRock) [inline packages/application-generic/src/health/completed-jobs-metric-queue.health-indicator.ts:13]: I don't see that this one `CompletedJobsMetricQueueServiceHealthIndicator` and `ActiveJobsMetricQueueServiceHealthIndicator` is used anywhere. Will we use them in the next PR?

Comment (Cliftonz) [reply packages/application-generic/src/health/completed-jobs-metric-queue.health-indicator.ts:13]: What do you mean by this?
I already have better uptime cron health attached to this.

Comment (LetItRock) [reply packages/application-generic/src/health/completed-jobs-metric-queue.health-indicator.ts:13]: @Cliftonz 
I see that there is no code that is using these two classes. They are imported in the `packages/application-generic/src/modules/queues.module.ts` but not used.

**Review by LetItRock — APPROVED** [review]
🥇

**Pull request merged**